src/EditorFeatures/CSharpTest/PullMemberUp/CSharpPullMemberUpTests.cs

```diff
@@ -2956,25 +2950,28 @@ public class Base
 }
         </Document>
         <Document FilePath = ""File2.cs"">
+<![CDATA[
 using System.Linq;
 
 public class Derived : Base
 {
-    public var ran[||]ge = Enumerable.Range(0, 5);
+    public IEnumerable<int> ran[||]ge = Enumerable.Range(0, 5);
```

Comment:
were these bugs before that are now fixed?

Reply:
These tests are checked in for this PR https://github.com/dotnet/roslyn/pull/54455
The original idea I think is to add System.Linq to the destination when try to move Enumerable.Range()
But the test case has syntax error

Reply:
please don't change existing test inputs.  Instead, update the results accordingly.  If you want teh test to test the new input, please create a new test.

Reply:
This test look strange because it's declaring a field using 'var'?
I think it is passing because of originally we reusing the field declaration.
Also found some other tests also assuming to reusing the field declaration, let me see if I can modify the code to satisfy these tests
https://sourceroslyn.io/#Microsoft.CodeAnalysis.EditorFeatures.UnitTests/CodeGeneration/CodeGenerationTests.CSharp.cs,1394

Reply:
Sigh : (
There several tests failing,
the 1st is this, because the syntax error here,
public class Bar
{
    public var range = Enumerable.Range(0, 5);
}

The type of this field is 'ErrorType', so the generated type syntax can just be 'Object'
It is passed originally because the old impl is reusing the FieldDeclaration completely.

2nd is
https://sourceroslyn.io/#Microsoft.CodeAnalysis.EditorFeatures.UnitTests/CodeGeneration/CodeGenerationTests.CSharp.cs,1394
To make this test pass, we need to reuse the leading comments.

3rd is
https://sourceroslyn.io/#Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests/ExtractClass/ExtractClassTests.cs,504
To make this test pass, we need to reuse the modifier. (because the default accessibility level is private, so from the symbol we don't know if the 'private' modifier is needed)

I put all the 'reusing' logic into the 'GetReuseableSyntaxNodeForSymbol' method,
you might want to recheck the current impl to see if it make sense to you.